### ISSUE TITLE
[Proposal] Объединить ксенобиологов и саентистов

### ISSUE THREAD

**Issue by Sakuya-Izayoi** [TauCetiStation/TauCetiClassic#6714]
Предлагаю выпилить отдельную профу ксенобиолога и либо забыть про них, либо сделать подпрофой саентиста с одним и тем же доступом, как это сделали с форонщиками. Мотивация, я думаю, понятна. Ксенобиологи сейчас могут только сидеть в своей душной лабе и ничего больше не делать, а простые саентисты которые иногда просто нуждаются в очках в эту лабу доступа не имеют. Получается ксенобиологи в отсутствии саентистов могут только дрочить слизней, а саентисты в отсутствии ксенобиологов вынуждены искать другие способы получения очков.

**Comment by ViDLiathan** [comment]
Да, давайте сделаем ещё меньше слотов для адекватных учёных либо же больше драк за работу с компом в рнд.

**Comment by Sakuya-Izayoi** [comment]
А зачем меньше слотов? Слотов всегда можно добавить, надо просто сделать всем учёным одинаковый доступ по всему рнд (об этом и пропозал).

**Comment by UDaV73rus** [comment]
Таким образом на тг заходил на ученых как баласт бесполезный, чтобы просто потыкать в слизней. Нужно другое решение
